packages/sdk/src/transactions/types.ts

```diff
@@ -95,7 +95,7 @@ export type Vin = {
   /**
    * Hex-encoded witness data
    */
-  txinwitness: string[];
+  txinwitness?: string[];
```

Comment:
i think it's better to make empty array on ordexer side instead of changing the type definition